### PR TITLE
MAM-4138-recursive-quote-indicator-above-images

### PR DESCRIPTION
**Pull request by derspyy**

### CODE REVIEW

File: Mammoth/Views/Cells/PostCardCell/PostCardQuotePost.swift [hunk 1]
```diff
@@ -744,7 +744,7 @@ fileprivate class PostCardQuoteIndicator: UIStackView {
         self.spacing = 8.0
         
         self.isLayoutMarginsRelativeArrangement = true
-        self.directionalLayoutMargins = NSDirectionalEdgeInsets(top: 4, leading: 0, bottom: 0, trailing: 0)
+        self.directionalLayoutMargins = NSDirectionalEdgeInsets(top: 4, leading: 12, bottom: 0, trailing: 0)
```
Comment (bnolens) [inline Mammoth/Views/Cells/PostCardCell/PostCardQuotePost.swift:747]: Is this intentional?

**Review by bnolens — CHANGES_REQUESTED** [review]
Please remove my note

File: Mammoth/Views/Cells/PostCardCell/PostCardQuotePost.swift (hunk 1 above)
Comment (derspyy) [reply Mammoth/Views/Cells/PostCardCell/PostCardQuotePost.swift:747]: yes, it's because we're not using the contentStack padding.

Comment (bnolens) [reply Mammoth/Views/Cells/PostCardCell/PostCardQuotePost.swift:747]: gotcha

**Review by bnolens — APPROVED** [review]
LGTM

**Pull request merged**